### PR TITLE
Use Valkyrie-Sequel

### PR DESCRIPTION
**Pull request by tpendragon**
Mostly a proof of concept. This should roughly half access and conversion times for many of our objects.

### CODE REVIEW

**Comment by tpendragon** [comment]
:laughing: Yes.

File: app/queries/count_members.rb [hunk 1]
```diff
@@ -39,6 +41,6 @@ def find_members_with_resource_query(resource, model)
   end
 
   def id_type
-    @id_type ||= orm_class.columns_hash["id"].type
+    @id_type ||= :uuid
```
Comment (hackmastera) [inline app/queries/count_members.rb]: probably doesn't need to be memoized anymore

**Comment by hackmastera** [comment]
https://github.com/samvera-labs/valkyrie-sequel

**Comment by tpendragon** [comment]
Going to see if instead of accepting arguments to build a Sequel connection if it's better to just have the argument for the adapter -be- a Sequel connection.

**Comment by tpendragon** [comment]
To test impact on real resources later, `https://figgy.princeton.edu/concern/scanned_resources/4058f68a-3ce6-475d-b61e-60625d0be56c/manifest` currently takes 1.9 seconds to render.

File: app/queries/find_saved_ids.rb [hunk 2]
```diff
@@ -12,7 +12,10 @@ def initialize(query_service:)
   end
 
   def find_saved_ids(ids:)
-    orm_class.where(id: ids.map(&:to_s)).pluck(:id).map do |id|
+    ids = ids.map(&:to_s).select do |id|
+      Valkyrie::Sequel::QueryService::ACCEPTABLE_UUID.match?(id)
```
[the remark this answers is not in the document]
Comment (tpendragon) [reply app/queries/find_saved_ids.rb]: This is actually necessary to pass the tests - it's behavior that ActiveRecord does silently.

**Pull request merged**